src/metpy/calc/thermo.py

```diff
@@ -46,6 +46,10 @@ def relative_humidity_from_dewpoint(temperature, dewpoint):
     `pint.Quantity`
         Relative humidity
 
+    Notes
+    _____
+
```

Comment:
```suggestion
    Notes
    -----
```
Almost done making the robots happy! If you can change these `_____` underscores to `-----` hyphens and remove the blank line (51) after these hyphens, this will be good to go!